### PR TITLE
Reportback fixes part 3a

### LINKED ISSUE #3808: Markup & logic for prior reportback submissions in Prove It form.
Establish the markup and logic for showing the appropriate interface elements on the Prove It reportback form when page initially loads.

### PR DESCRIPTION
**Pull request by weerd**
Resolves #3808

Wrapping up most of the markup and logic when campaign loads with/or without having user reportbacks in Prove It section.

If there are no reportbacks from the user, then show the large button (which still needs further styling):
screen shot 2015-01-27 at 12 27 49 pm

Otherwise, if there are prior reportback submissions, then show the last submission prominently, allow user to "Add another photo" (file upload dialogue), and show prior submissions:
screen shot 2015-01-27 at 12 27 11 pm

@aaronschachter @sbsmith86 
CC: @barryclark @DFurnes

### CODE REVIEW

**Comment by weerd** [comment]
FYI: More styling needs to happen, so kind of ignore some of the funkyness and lack of whitespace etc, for now ;) huggles!

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc [hunk 1]
```diff
@@ -53,26 +54,76 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
...
   if (!isset($entity->rbid)) {
     $entity->rbid = 0;
+
+    $form['reportback_submissions']['reportback_file']['#attributes']['is_supersized'] = TRUE;
+
     $submit_label = t("Submit your pic");
   }
-  // Else, it's update form.
+
+  // Else, it's update an form.
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc]: An update form

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc [hunk 2]
```diff
@@ -378,4 +422,4 @@ function dosomething_reportback_flag_confirm_submit($form, &$form_state) {
   }
   // Log the flag operation.
   $reportback->insertLog($op);
-}
+}
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc:474]: :cactus:

File: lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php [hunk 3]
```diff
@@ -0,0 +1 @@
+<?php print $image; ?>
```
Comment (sbsmith86) [inline lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php:1]: new line?

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc (hunk 1 above)
Comment (weerd) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc]: Lol. I had it fixed correctly before and then Vagrant crapped out on me and I had to redo some code. Put the 'an' in the wrong place, haha.

**Comment by aaronschachter** [comment]
Looks ok, assuming this has been tested and works when the relevant theme/module settings are turned off and this is safe to deploy without turning on.

File: lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php (hunk 3 above)
Comment (weerd) [inline lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php:1]: This is just for the template in the module, that doesn't get used cause it's overridden in the theme ;)

**Comment by weerd** [comment]
Oh, good call. Lemme check on that.

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc (hunk 2 above)
Comment (weerd) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc:474]: Not sure what this was. My editor being funky?

**Comment by weerd** [comment]
@aaronschachter Ok! Updated to account for feature flags. I had to use a large `if/else` conditional for part of the form to hide the new approach from Production. I added comments and an esaily findable `@TODO_RBv2` comment to know where things need to be edited and removed post-launch!

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc (hunk 2 above)
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc:474]: Looks like you removed the blank line, https://www.drupal.org/node/318#indenting

File: lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php (hunk 3 above)
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_reportback/theme/reportback-latest-submission.tpl.php:1]: Same here

**Comment by aaronschachter** [comment]
Nice PHP :+1:  Feel free to fix line endings in subsequent PR to get this out.

**Pull request merged**